Shared/Samples/Edit feature attachments/EditFeatureAttachmentsView.Model.swift

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension EditFeatureAttachmentsView {
+    // MARK: - Model
+    
+    @MainActor
+    class Model: ObservableObject {
+        let map = Map(basemapStyle: .arcGISOceans)
+        
+        /// The currently selected map feature.
+        private var selectedFeature: ArcGISFeature?
+        
+        /// The placement of the callout on the map.
+        var calloutPlacement: CalloutPlacement?
+        
+        /// Holds the attachments of the currently selected feature.
+        @Published var attachments: [Attachment] = []
+        
+        /// The text shown on the callout.
+        @Published var calloutText = ""
+    
+        /// The text shown on the callout.
+        @Published var calloutDetailText = ""
+        
+        /// A Boolean value that indicates whether the callout placement should be offset for the map magnifier.
+        @Published var calloutShouldOffset = false
+        
+        /// The `FeatureLayer` populated with data by the feature table using the remote service url. 
```

Comment:
```suggestion
        /// The feature layer populated with data by the feature table using the remote service url. 
```
The documentation should describe what the value is not the type.